.circleci/config.yml

```diff
@@ -151,11 +151,13 @@ references:
     run:
       name: Create build tag
       command: |
-        current_branch=$(git branch | grep \* | cut -d ' ' -f2)
-        short_version=$(git rev-parse --short $current_branch)
-        build_tag=cmoj-${current_branch}-${short_version}
-        echo "export BUILD_TAG=${build_tag}" >> $BASH_ENV
+        prefix="ct"
```

Comment:
Small point; could `ct` be inserted into the string on line 156 instead of being assigned to a variable?

Reply:
It could, the reason I did it this way is that we will have similar code across a few applications and I wanted it to be obvious that the prefix would need to change if this code is re-used elsewhere.